lxqt-config-appearance/lxqtthemeconfig.ui

```diff
@@ -51,6 +51,13 @@
      </column>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="wallpaperOverride">
+     <property name="text">
+      <string>Override independently set wallper</string>
```

Comment:
@palinek you're right. Maybe "Override user-defined wallpaper when selecting themes".